packages/utils/src/fhir-deduplication/dedup-files.ts

```diff
@@ -56,6 +57,11 @@ async function main() {
       } resources in ${elapsedTimeFromNow(startedAt)} ms.`
     );
 
+    const resources =
+      resultingBundle.entry?.map(entry => entry.resource).filter((r): r is Resource => !!r) ?? [];
+    const isValid = validateReferences(resources, logsFolderName);
+    console.log(`Reference validation result: ${isValid ? "Valid" : "Invalid"}`);
```

Comment:
no. Its helpful

Reply:
this might be helpful during development, but i dont see how this could be helpful in prod. If you want to log it, you should use log(), so there's context attached to it, no?

Reply:
its a util file

Reply:
lol didnt notice. all good